security/rbac/scripts/submit_datagen_pageviews_config_avro.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+CONNECT_HOST=localhost
+
+if [[ $1 ]];then
+    CONNECT_HOST=$1
+fi
+
+HEADER="Content-Type: application/json"
+DATA=$( cat << EOF
+{
+  "name": "datagen-pageviews",
+  "config": {
+    "connector.class": "io.confluent.kafka.connect.datagen.DatagenConnector",
+    "kafka.topic": "pageviews",
+    "quickstart": "pageviews",
+    "key.converter": "org.apache.kafka.connect.storage.StringConverter",
+    "value.converter": "io.confluent.connect.avro.AvroConverter",
+    "value.converter.schema.registry.url": "http://localhost:8081",
+    "value.converter.basic.auth.credentials.source": "USER_INFO",
+    "value.converter.basic.auth.user.info": "client:client1",
```

Comment:
These credentials should match the ones in the `producer.override.sasl.jaas.config`, so that there's one service account to track for the connector that deals with both Kafka interactions (writing to, reading from, and creating topics) and Schema Registry interactions (writing to and reading from subjects).

Reply:
One SA for both the connector and SR makes logical sense, but is that the way it will always work?  It may be different, right?, depending on how SA creds are handed out?

Reply:
While it's certainly possible, I don't think it's likely, and it goes against the recommended flow of one SA per connector. I think we should demonstrate the common case here, but I'll leave that up to you.

Reply:
I'm changing everything from `connector` to `client`.